.github/workflows/runnable_cxx.yml

```diff
@@ -78,7 +80,27 @@ jobs:
           - { os: ubuntu-18.04, target: msvc-2017 }
           - { os: ubuntu-18.04, target: msvc-2015 }
           - { os: ubuntu-18.04, target: msvc-2013 }
+          - { os: ubuntu-20.04, target: msvc-2019 }
+          - { os: ubuntu-20.04, target: msvc-2017 }
+          - { os: ubuntu-20.04, target: msvc-2015 }
+          - { os: ubuntu-20.04, target: msvc-2013 }
+          # 18.04 only has g++-5 through to 9, and clang-8.0.0 through to 10.0.0
```

Comment:
Note that the reason for going with exclude list no longer holds: In the early day, AFAIK, Github didn't support including dimensions using `include`, so I had to create that huge matrix and exclude things. Last I tried, it worked though, which was a while ago. But we can leave this improvement for later.